package-res/resources/web/test/util/TextFormatterSpec.js

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2010 - 2015 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+define([ "common-ui/util/TextFormatter" ], function(TextFormatter) {
+
+  describe("TextFormatter", function() {
+
+    describe("getFormatType", function() {
+      var typeMap = {
```

Comment:
Do you want to be able to make it so that the typeMap is retrievable from the class, in case it changes?